manifests/puppet.pp

```diff
@@ -34,35 +36,39 @@
     ca            => $default_ca,
     generate      => $generate,
     regenerate    => $regenerate,
-    deploy        => $deploy,
+    deploy        => false,
     password_file => $ca_key_password_file,
     build_dir     => $certs::ssl_build_dir,
   }
 
   if $deploy {
     file { "${pki_dir}/puppet":
       ensure => directory,
-      owner  => 'puppet',
+      owner  => $owner,
+      group  => $group,
       mode   => '0700',
```

Comment:
This breaks things since Puppet is longer able to read these files:
```suggestion
      owner  => $owner,
      group  => $group,
      mode   => '0750',
```